docs/charmed_k8s_pod.md

```diff
@@ -0,0 +1,93 @@
+## Working with Charmed Kubernetes From within a pod
+
+### Setup
+
+Assuming you have juju and charmed kubernetes already setup, let's discuss the steps to launch Spark jobs from within a pod in charmed kubernetes.
+
+First thing is to get the kubeconfig of the charmed kubernetes setup.
+
+```shell
+$ juju scp kubernetes-control-plane/0:config ~/.kube/config
+```
+
+Now we launch a pod using Canonical Data Plarform's OCI image for Apache Spark.
+
+Create a pod manifest ```shell-demo.yaml``` for use with charmed kubernetes, something like this
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: shell-demo
+  namespace: spark-test-ns
+spec:
+  containers:
+  - name: spark-client
+    image: dataplatformoci/spark:3.3.2
```

Comment:
is this the right image now? I thought we were moving these? @deusebio

Reply:
Yes, but we should first set up the GHCR. The directive here (from Jon)  is to keep - for the time being - whatever we have in dockerhub there. if we start a new project, that should however land in GHCR. But to do so, I suppose @taurus-forever should first set up the GHCR. 

Once we have a GHCR, we can then put a task in the backlog to migrate the images. It should be fairly easy to do